src/features/game/types/crops.ts

```diff
@@ -15,17 +17,16 @@ export type CropName =
   | "Wheat"
   | "Kale";
 
-export type Crop = {
+export interface Crop extends ItemDetails {
   sellPrice: Decimal;
   harvestSeconds: number;
   name: CropName;
-  description: string;
-};
+}
 
 /**
  * Crops and their original prices
  */
-export const CROPS: () => Record<CropName, Crop> = () => ({
+export const CROPS: Record<CropName, Crop> = {
```

Comment:
The func is needed in this case to ensure that we always get the current marketRate.

Every time we access `CROPS()` we need to calculate the cost at run time.

This means if someone is playing during a halvening event, the prices update automatically. With the func removed, these values will be set at initialisation time and a user will be interacting with the incorrect prices